### DISCUSSION TITLE
New Contributor

### DISCUSSION THREAD

**Discussion by Aman123lug** [cncf/mentoring#746] [Q&A]
Hy, everyone i am new in CNCF contributor can anyone guide me how can i learn something new about CNCF. I want to Contribute in GSOC this is my first open source

**Comment by money8203** [comment]
Hi @Aman123lug!

If you want to be a Cloud Native contributor, I suggest starting by checking out the [CNCF Contribution](https://github.com/cncf/mentoring/blob/main/summerofcode/README.md) page.

If you're interested in being a mentee, you can also take a look around this repo and see what opportunities there are. You will find all the info regarding GSOC + CNCF in this repo.

**Reply by Aman123lug** [reply]
Ohh okay thanks
